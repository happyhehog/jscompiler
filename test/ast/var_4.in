var a, b = 3, c;