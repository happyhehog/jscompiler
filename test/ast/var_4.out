ProgramNode (1:0, 1:15)
    VariableDeclarationListNode (1:4, 1:14)
    --> declarations:
        VariableDeclarationNode (1:4, 1:4)
        --> identifier:
            IdentifierNode (1:4, 1:4) : { name: a }
        VariableDeclarationNode (1:7, 1:11)
        --> identifier:
            IdentifierNode (1:7, 1:11) : { name: b }
        --> initValue:
            LiteralNode (1:11, 1:11) : { value: 3 }
        VariableDeclarationNode (1:14, 1:14)
        --> identifier:
            IdentifierNode (1:14, 1:14) : { name: c }